cmd/syslog/syslog.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"gopkg.in/mcuadros/go-syslog.v2"
+)
+
+// The daemon process logs events using Go's log/syslog package. That
+// package, when using unix domain sockets, expects one of "/dev/log",
+// "/var/run/syslog", or "/var/run/log" so we bind a matching path.
+var listenAddress string = "/var/run/syslog"
+
+func main() {
+	channel := make(syslog.LogPartsChannel)
+	handler := syslog.NewChannelHandler(channel)
+
+	server := syslog.NewServer()
+	server.SetFormat(syslog.RFC3164)
+	server.SetHandler(handler)
+
+	if val := os.Getenv("DAEMON_SYSLOG_ADDRESS"); val != "" {
```

Comment:
I don't see the env var defined anywhere is this used to export syslog externally instead of to /var/run/syslog ? is just and address or address:port ?

Reply:
I will remove this as I used it for debugging only.